### DISCUSSION TITLE
Associating link with text in Text Widget

### DISCUSSION THREAD

**Discussion by vikash0705** [mum4k/termdash#325] [General]
Is there a way that inside text widget can we show links/url clicking on which it navigates to that page in browser, or can we map a word to a link inside text widget?

**Comment by mum4k** [comment]
Thank you for reaching out @vikash0705. I might be wrong, but I think that the interpretation of links into clickable actions happens at the terminal application rather than at the termdash level.

While termdash can display any text, including links, it is up to the terminal to be able to react to them and open a browser.

**Reply by vikash0705** [reply]
Thanks @mum4k . Do we have any way of applying padding in a text widget on all sides?

**Reply by mum4k** [reply]
Margin and padding can be applied at the container level, i.e. it can be applied to all widgets. Does this help?

https://github.com/mum4k/termdash/wiki/Margin-and-padding
